flocker/cli/functional/test_deploy_script.py

```diff
@@ -69,7 +69,7 @@ def test_installs_public_sshkeys(self):
             nodes=frozenset([
                 Node(
                     hostname=str(self.server.ip),
```

Comment:
And these should really all be `unicode` instead of `str` (or, like, neither... structured objects for the win).  :question: